src/Menu/HomePageRenderer.cpp

```diff
@@ -0,0 +1,449 @@
+	io.WantCaptureMouse = true;
+	io.WantCaptureKeyboard = true;
+	io.MouseDrawCursor = true;  // Show ImGui cursor
+
+	// Center the window properly with rounded corners and thin border
+	ImVec2 center = ImVec2(io.DisplaySize.x * 0.5f, io.DisplaySize.y * 0.5f);
+	ImGui::SetNextWindowPos(center, ImGuiCond_Always, ImVec2(0.5f, 0.5f));
+	ImGui::SetNextWindowSize(ImVec2(500, 350), ImGuiCond_Always);
+
+	// Style for rounded window with thin border
+	ImGui::PushStyleVar(ImGuiStyleVar_WindowRounding, 8.0f);
+	ImGui::PushStyleVar(ImGuiStyleVar_WindowBorderSize, 1.0f);
+
+	ImGuiWindowFlags flags = ImGuiWindowFlags_NoResize | ImGuiWindowFlags_NoMove |
+	                         ImGuiWindowFlags_NoCollapse | ImGuiWindowFlags_NoSavedSettings |
+	                         ImGuiWindowFlags_NoScrollbar | ImGuiWindowFlags_NoTitleBar;  // Prevent scrolling and remove title
+
+	if (!ImGui::Begin("##FirstTimeSetup", nullptr, flags)) {
+		ImGui::PopStyleVar(2);
+		ImGui::End();
+		return;
+	}
+
+	auto menu = Menu::GetSingleton();
+
+	// Render CS logo as background watermark with proper aspect ratio
+	if (menu && menu->uiIcons.logo.texture) {
+		ImVec2 windowPos = ImGui::GetWindowPos();
+		ImVec2 windowSize = ImGui::GetWindowSize();
+
+		// Get the original texture size to maintain aspect ratio
+		ImVec2 textureSize = menu->uiIcons.logo.size;
+		float aspectRatio = textureSize.x / textureSize.y;
+
+		// Set desired height and calculate width to maintain aspect ratio
+		float logoHeight = 260.0f;
+		float logoWidth = logoHeight * aspectRatio;
+
+		ImVec2 logoMin(windowPos.x + (windowSize.x - logoWidth) * 0.5f,
+			windowPos.y + (windowSize.y - logoHeight) * 0.5f);
+		ImVec2 logoMax(logoMin.x + logoWidth, logoMin.y + logoHeight);
+
+		// Render as subtle watermark background
+		ImU32 watermarkColor = IM_COL32(255, 255, 255, 60);
+		ImGui::GetWindowDrawList()->AddImage(menu->uiIcons.logo.texture, logoMin, logoMax,
+			ImVec2(0, 0), ImVec2(1, 1), watermarkColor);
+	}
+
+	// Center all content
+	float windowWidth = ImGui::GetWindowWidth();
+
+	// Welcome title - centered
+	const char* welcomeTitle = "Welcome to Community Shaders!";
+	float welcomeTitleWidth = ImGui::CalcTextSize(welcomeTitle).x;
+	ImGui::SetCursorPosX((windowWidth - welcomeTitleWidth) * 0.5f);
+	ImGui::Text("%s", welcomeTitle);
+
+	// Version text - centered
+	const char* versionText = "This appears to be a new install, update, or reinstallation of Community Shaders.";
+	float versionWidth = ImGui::CalcTextSize(versionText).x;
+	ImGui::SetCursorPosX((windowWidth - versionWidth) * 0.5f);
+	ImGui::Text("%s", versionText);
+
+	ImGui::Spacing();
+
+	// Description - centered
+	const char* description = "Please select a hotkey to access the menu:";
+	float descWidth = ImGui::CalcTextSize(description).x;
+	ImGui::SetCursorPosX((windowWidth - descWidth) * 0.5f);
+	ImGui::Text("%s", description);
+
+	// Hotkey selection - centered
+	static int selectedKey = 0;
+	const char* keyOptions[] = {
```

Comment:
How does this interact with the defaults we have? You've also introduced another set of potential keys we need to keep up to date in another file. You should centralize all the special keys we need in one location. Also, I'm not sure you need to define key names. Since these are predefined windows keys, there's probably an existing enum to use instead of manually creating a list.